best-practices-zarr.md

```diff
@@ -0,0 +1,549 @@
+
+- STAC object types suitable for Zarr stores
+- Asset hierarchy and organization for Zarr groups and arrays
+- Discovery patterns for variables within multidimensional stores
+- Integration with datacube extension for variable and dimension metadata
+- Use of link relationships to reference Zarr stores
+- Asset metadata for multi-resolution and hierarchical data structures
+
+The following specifications and extensions are referenced in this document:
+
+- [STAC v1.1](https://github.com/radiantearth/stac-spec/tree/v1.1.0)
+- [Datacube Extension v2.x](https://github.com/stac-extensions/datacube)
+- [Raster Extension v2.x](https://github.com/stac-extensions/raster)
+- [CF Extension v1.x](https://github.com/stac-extensions/cf)
+- [Electro Optical (EO) Extension v2.x](https://github.com/stac-extensions/eo)
+- [Projection Extension v2.x](https://github.com/stac-extensions/projection)
+
+## Glossary
+
+### Zarr Terminology
+
+See the [Zarr specification](https://zarr-specs.readthedocs.io/en/latest/v3/core/#concepts-and-terminology) for complete terminology.
+
+Key terms:
+
+- **Zarr Store**: A storage system that can hold a Zarr hierarchy (e.g., a directory, zip file, or object store prefix) conforming to the [Zarr abstract store interface](https://zarr-specs.readthedocs.io/en/latest/v3/core/index.html#abstract-store-interface)
+- **Group**: Any node - including the root node - in a Zarr hierarchy that may have child nodes.
+- **Array**: A leaf node in a Zarr hierarchy that represents a multidimensional array with chunked storage
+- **Chunk**: A contiguous region of an array that is stored as a single object
+
+Conventions terms:
+
+- **Multiscales Group**: A [convention](https://github.com/zarr-conventions/multiscales) for organizing multi-resolution data in Zarr stores, typically using a tree structure where each level represents a different resolution.
+
+### Xarray/NetCDF/CF-Specific Concepts
+
+- **Variable**: A named data array within a Zarr group, typically representing a measured or derived quantity
+- **Dimension**: An axis of a multidimensional array (e.g., x, y, time, band)
+- **Coordinate Variable**: An array that provides coordinate values along a dimension
+- **Data Variable**: An array containing measured or derived data values
+
+## General Rules
+
+### STAC Object Type
+
+Zarr stores and n-dimensional arrays can be represented by 2 STAC object types depending on the use case.
+
+#### STAC Item
+
+A STAC Item is appropriate when the Zarr store or group represents a single logical dataset or scene, such as:
+
+- A satellite scene with multiple bands and resolutions
+- A time-specific climate model run output with multiple variables and dimensions
+- A single time slice of a time series
+- A single multiscale dataset
+
+It is **HIGHLY RECOMMENDED** to reference a unique Zarr store per STAC Item to avoid ambiguity when accessing data via the asset hrefs.
```

Comment:
This meaning of this line isn't clear to me.

Reply:
The reference is the recommended `link` to the zarr store. The idea is to discourage having Stac Item with assets pointing to 2 different Zarr store.